app/build.gradle

```diff
@@ -54,4 +54,4 @@ dependencies {
     implementation('com.google.android.gms:play-services-base:11.8.0') {
```

Comment:
You should also remove the reference from:
`proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'`